src/object/task/taskbuild.cpp

```diff
@@ -392,15 +411,17 @@ Error CTaskBuild::Start(ObjectType type)
     err = FlatFloor();
     if ( err != ERR_OK )  return err;
 
+    pv = m_object->GetPosition();
+    pm = m_metal->GetPosition();
+    if(!m_physics->GetLand() && abs(pm.y-pv.y)>8.0f) return ERR_BUILD_METALAWAY;
+    
```

Comment:
Linter issue: `using integer absolute value function 'abs' when argument is of floating point type`